CMakeLists.txt

```diff
@@ -427,6 +421,12 @@ if(NOT WIN32)
 	)
 endif()
 
+# changelog
+add_custom_target(changelog
+	COMMAND git log | sed '/^commit/d\; /^Merge/d' > "${CMAKE_BINARY_DIR}/ChangeLog"
```

Comment:
hardcoded commands, you need to use find_program and find git and sed. This is not an issue on Linux because of standard paths but it would be an issue on OSX and especially Windows

Reply:
Wouldn't it be better to put it inside `if(NOT WIN32)` ?